e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -891,13 +891,62 @@ describe("scenarios > embedding > full app", () => {
         () => {
           H.restore("postgres-12");
           cy.signInAsAdmin();
+          H.createModelFromTableName({
+            tableName: "orders",
+            modelName: "Orders Model (Postgres)",
+          });
           startNewEmbeddingQuestion({ isMultiStageDataPicker: true });
           selectTable({ tableName: "Orders", databaseName: "QA Postgres12" });
           clickOnDataSource("Orders");
           verifyTableSelected({
             tableName: "Orders",
             databaseName: "QA Postgres12",
           });
+
+          cy.log(
+            "assert that even after selecting a data source from one database, the data picker still shows the other data sources database",
```

Comment:
This asserts the data picker after we've selected a data source already i.e. clicking the same starting data source picker.

![image](https://github.com/user-attachments/assets/e8762355-82da-44f8-bd5d-9701aa06c85a)